nemo/collections/asr/parts/submodules/causal_convs.py

```diff
@@ -148,7 +148,9 @@ def update_cache(self, x, cache=None, cache_next=None):
             x = torch.cat((needed_cache, x), dim=-1)
 
         if cache_next is not None:
-            input_x_kept = input_x[:, :, : input_x.size(-1) - self.cache_drop_size]
+            input_x_size = torch.tensor(input_x.size(-1) - self.cache_drop_size, dtype=torch.int64)
+            input_x_size = input_x_size.clip(min=1, max=input_x.size(-1))
+            input_x_kept = input_x[:, :, :input_x_size]
```

Comment:
Would you please check it with export nemo to make sure the ONNX conversion is still working?